packages/core/src/Editor.ts

```diff
@@ -235,7 +238,17 @@ export class Editor extends EventEmitter<EditorEvents> {
    * Creates an extension manager.
    */
   private createExtensionManager(): void {
-    const coreExtensions = this.options.enableCoreExtensions ? Object.values(extensions) : []
+
+    const coreExtensions = this.options.enableCoreExtensions ? [
```

Comment:
Since we only have less than 10 core extensions I'd think it's okay to just load them manually here – that way we can pass configurations down to them.